tyr/servers/server.py

```diff
@@ -199,6 +200,41 @@ def configure(self):
         self.log.info('Using Chef path "{path}"'.format(
                                 path = self.chef_path))
 
+        if self.dns_zones is None:
+            self.log.warn('No DNS zones specified')
+            self.dns_zones = [
+                {
+                    'id': {
```

Comment:
So this hardcodes it to our zone ids. Is this something you could easily move to a config file?